package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "screwdriver-api",
-  "version": "0.5.0",
+  "version": "1.0.0",
```

Comment:
With all of these changes, I think we should bump major version for this repo 👍 
We're currently at v0.5.975